src/Error/UserError.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace GraphQL\Error;
+
+/**
+ * Class UserError
+ *
+ * Note:
+ * Error that can be display safely to client...
+ *
+ * @package GraphQL\Error
+ */
+class UserError extends InvariantViolation
```

Comment:
extends `InvariantViolation` to avoid BC, next major version we can change this.